usaspending_api/references/constants.py

```diff
@@ -0,0 +1,2 @@
+TOTAL_BUDGET_AUTHORITY = 8361447130497.72
+TOTAL_OBLIGATIONS_INCURRED = 4690484214947.31
```

Comment:
`TOTAL_OBLIGATIONS_INCURRED` added but not used in the application.

Reply:
Yup, added it in case we have issues with any calculations around that. It was just a preemptive attempt to solve a problem that hasn't happened yet.